portal-api/api-tests/api-test-users.js

```diff
@@ -279,6 +279,37 @@ const setUpApiTestUser = async (as) => {
   return { token, ...userToCreate };
 };
 
+const newAdminTestUser = {
+  username: 'new-admin-tester@ukexportfinance.gov.uk',
+  password: 'NewP@ssword1234',
+  firstname: 'NewAdmin',
+  surname: 'Test User',
+  email: 'new-admin-tester@ukexportfinance.gov.uk',
+  timezone: 'Europe/London',
+  roles: [ADMIN],
+  bank: banks.any,
+};
+
+const setUpNewAdminTestUser = async (as) => {
```

Comment:
Should this be called `setUpAdminTestUser`? Since the other function is called `setUpApiTestUser`, rather than `setUpNewApiTestUser`

Reply:
This is already called as setUpAdminTestUser. Not sure if I misunderstood.

Reply:
setUpAdminTestUser please.

Reply:
Changed to setUpAdminTestUser.